validate_test.go

```diff
@@ -347,3 +348,11 @@ func cumSumSuccess(_ context.Context, stream *connect.BidiStream[calculatorv1.Cu
 func cumSumError(_ context.Context, _ *connect.BidiStream[calculatorv1.CumSumRequest, calculatorv1.CumSumResponse]) error {
 	return connect.NewError(connect.CodeInternal, errors.New("boom"))
 }
+
+func fieldPath(field *validatepb.FieldPath) string {
```

Comment:
Can this be replaced with [protovalidate.FieldPathString](https://pkg.go.dev/github.com/bufbuild/protovalidate-go#FieldPathString).